python/mxnet/visualization.py

```diff
@@ -134,17 +134,23 @@ def print_layer_summary(node, out_shape):
                             pre_filter = pre_filter + int(shape[0])
         cur_param = 0
         if op == 'Convolution':
-            if ("no_bias" in node["attrs"]) and int(node["attrs"]["no_bias"]):
-                cur_param = pre_filter * int(node["attrs"]["num_filter"])
+            if ("no_bias" in node["attrs"]) and node["attrs"]["no_bias"] == 'True':
+                num_group = int(node["attrs"]["num_group"]) if \
+                   ("num_group" in node["attrs"]) else 1
+                cur_param = (pre_filter * int(node["attrs"]["num_filter"])) \
+                   // num_group
```

Comment:
the outer parentheses are not necessary and make it hard to read. would you clean it up?